include/swift/AST/DiagnosticsParse.def

```diff
@@ -686,6 +686,16 @@ ERROR(sil_witness_assoc_conf_not_found,none,
 ERROR(sil_witness_protocol_conformance_not_found,none,
       "sil protocol conformance not found", ())
 
+// SIL differentiability witnesses
+ERROR(sil_diff_witness_expected_keyword,PointsToFirstBadToken,
```

Comment:
Keyword has a specific meaning in SIL syntax. Use "token" instead.
```suggestion
ERROR(sil_diff_witness_expected_token,PointsToFirstBadToken,
```

Reply:
Done in https://github.com/apple/swift/pull/27487/commits/2073f863826fb305c1fc105a520d84de1e1463ec.